src/dotfilepatcher.cpp

```diff
@@ -562,6 +563,10 @@ static bool readSVGSize(const QCString &fileName,int *width,int *height)
     {
       found=true;
     }
+    else if (sscanf(line.c_str(),"<svg xmlns=\"http://www.w3.org/2000/svg\" xmlns:xlink=\"http://www.w3.org/1999/xlink\" width=\"%d\" height=\"%d\"",width,height)==2)
```

Comment:
When using sscanf the parameters should be &width and &height otherwise doxygen may crash

Reply:
No not in this case, `width` and `height` are from `static bool readSVGSize(const QCString &fileName,int *width,int *height)` also check a view lines above for the old test.

Reply:
ah indeed, I didn't see this important bit of context.